carbon-design-system/carbon
# feat(themes): add values for v11 tokens

Closes #8832

This PR adds in the set of tokens for v11. It also adds in a new preview, `preview-v11`, to interact with the token background groups and sets that we would like to include for this release.

#### Changelog

**New**

- Add token values for v11 in `next` to not conflict with any existing theme work
- Add new format to TokenFormat for formatting scss identifiers
- Add `preview-v11` for previewing v11 tokens in a new token explorer

**Changed**

**Removed**

#### Testing / Reviewing

- Verify all tests pass for themes (meaning that all tokens exist and are defined in each theme)
- Visit the deploy preview for elements
- Verify that the themes -> preview-v11 site renders correctly
- (For design) verify that tokens match the final token list in Notion
